### ISSUE TITLE
@match_info_schema results in swagger parameter entries for both "path" and "match_info"

### ISSUE THREAD

**Issue by waweber** [maximdanilchenko/aiohttp-apispec#74]
If you set up a schema that validates a route's `match_info`, the Swagger output will have a generated parameter description for both the parameters in the path and the ones in the schema.

Example, with a route that must have a `customer_id` that is a valid UUID:

```python3

class CustomerIdSchema(Schema):
    customer_id = fields.UUID(required=True)

@docs(...)
@routes.get("/customers/{customer_id}")
@match_info_schema(CustomerIdSchema)
async def route(request):
    customer_id = request.match_info["customer_id"]
    ...
```

The resulting Swagger schema will have a `parameters` property that looks like this:
```json
"parameters": [
          {
            "in": "match_info",
            "name": "customer_id",
            "required": true,
            "type": "string",
            "format": "uuid"
          },
          {
            "in": "path",
            "name": "customer_id",
            "required": true,
            "type": "string"
          }
        ],
```

It contains both the `customer_id` from the route path as well as from the schema. In the Swagger page it would ask you to enter both.

I think specifying `match_info_schema` should replace these parameters, right?

**Comment by waweber** [comment]
Follow up: doing
`@request_schema(CustomerIdSchema, locations=["path"], put_into="path")` results in expected behavior

**Comment by maximdanilchenko** [comment]
@waweber hi! https://github.com/maximdanilchenko/aiohttp-apispec/pull/73 opened by @maksimvrs  fixes it. But @maksimvrs doesn't answer for my question about this PR. So I think I will close it by myself soon.

**Comment by lafrech** [comment]
See https://github.com/marshmallow-code/apispec/pull/517.

And please comment on https://github.com/marshmallow-code/apispec/issues/587 if you have any insight.

**Comment by maximdanilchenko** [comment]
Closed with #115
